src/common/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { describe, test } from 'vitest';
+import { render } from '@testing-library/react';
+import Button from './Button';
+
+
+describe('<Button/>', () => {
+    test('full color Button ', () => {
+        render(<Button
+            onClick={() => console.log("succeed")}
+            isLink={false}
+            value="התחברות"
+            backgroundColor="#6503A6"
+            borderColor="#F2CB05"
+            border="1px solid black"
+            outlineColor="#F2B704"
+            color="white"
```

Comment:
מה בודקים הטסטים האלו?
טסט צריך לבדוק שקורה מה שהוא מצפה
בדרך כלל יש טסט של should render, שבודק רק שלא קורות שגיאות
ואז יש טסט של should emit click event
או should show button label וכו כדי לבדוק את כל מה שהקומפוננטה אמורה לעשות

Reply:
אני אשנה אותם אם יירצה ה'